packages/encoding/src/OnChainData.ts

```diff
@@ -18,6 +18,25 @@ export interface StarkExProgramOutput {
   conditions: PedersenHash[]
 }
 
+//
```

Comment:
The comment should probably not be empty :)

Reply:
Yup, fixed it